xrpl/rpc/client.go

```diff
@@ -120,6 +126,22 @@ func (c *Client) Submit(txBlob string, failHard bool) (*requests.SubmitResponse,
 	})
 }
 
+// SubmitTx signs the transaction (if needed) by using getSignedTx,
```

Comment:
Remove `getSignedTx`. These comments are uploaded to the package reference and should not expose private behavior (since `getSignedTx` is a private method)